keyboards/ghs/rar/info.json

```diff
@@ -0,0 +1,512 @@
+                {"x": 12, "y": 0},
+                {"x": 13, "y": 0},
+                {"x": 14, "y": 0},
+
+                {"x": 15.5, "y": 0},
+
+                {"x": 0, "y": 1.5},
+                {"x": 1, "y": 1.5},
+                {"x": 2, "y": 1.5},
+                {"x": 3, "y": 1.5},
+                {"x": 4, "y": 1.5},
+                {"x": 5, "y": 1.5},
+                {"x": 6, "y": 1.5},
+                {"x": 7, "y": 1.5},
+                {"x": 8, "y": 1.5},
+                {"x": 9, "y": 1.5},
+                {"x": 10, "y": 1.5},
+                {"x": 11, "y": 1.5},
+                {"x": 12, "y": 1.5},
+                {"x": 13, "y": 1.5, "w": 2},
+
+                {"x": 15.5, "y": 1.5},
+
+                {"x": 0, "y": 2.5, "w": 1.5},
+                {"x": 1.5, "y": 2.5},
+                {"x": 2.5, "y": 2.5},
+                {"x": 3.5, "y": 2.5},
+                {"x": 4.5, "y": 2.5},
+                {"x": 5.5, "y": 2.5},
+                {"x": 6.5, "y": 2.5},
+                {"x": 7.5, "y": 2.5},
+                {"x": 8.5, "y": 2.5},
+                {"x": 9.5, "y": 2.5},
+                {"x": 10.5, "y": 2.5},
+                {"x": 11.5, "y": 2.5},
+                {"x": 12.5, "y": 2.5},
+                {"x": 13.5, "y": 2.5, "w": 1.5},
+
+                {"x": 15.5, "y": 2.5},
+
+                {"x": 0, "y": 3.5, "w": 1.75},
+                {"x": 1.75, "y": 3.5},
+                {"x": 2.75, "y": 3.5},
+                {"x": 3.75, "y": 3.5},
+                {"x": 4.75, "y": 3.5},
+                {"x": 5.75, "y": 3.5},
+                {"x": 6.75, "y": 3.5},
+                {"x": 7.75, "y": 3.5},
+                {"x": 8.75, "y": 3.5},
+                {"x": 9.75, "y": 3.5},
+                {"x": 10.75, "y": 3.5},
+                {"x": 11.75, "y": 3.5},
+                {"x": 12.75, "y": 3.5, "w": 2.25},
+
+                {"x": 15.5, "y": 3.5},
+
+                {"x": 0, "y": 4.5, "w": 2.25},
+                {"x": 2.25, "y": 4.5},
+                {"x": 3.25, "y": 4.5},
+                {"x": 4.25, "y": 4.5},
+                {"x": 5.25, "y": 4.5},
+                {"x": 6.25, "y": 4.5},
+                {"x": 7.25, "y": 4.5},
+                {"x": 8.25, "y": 4.5},
+                {"x": 9.25, "y": 4.5},
+                {"x": 10.25, "y": 4.5},
+                {"x": 11.25, "y": 4.5},
+                {"x": 12.25, "y": 4.5, "w": 1.75},
+
+                {"x": 14.25, "y": 4.75},
+
+                {"x": 15.5, "y": 4.5},
+
+                {"x": 0, "y": 5.5, "w": 1.25},
+                {"x": 1.25, "y": 5.5, "w": 1.25},
+                {"x": 2.5, "y": 5.5, "w": 1.25},
+                {"x": 3.75, "y": 5.5, "w": 6.25},
+                {"x": 10, "y": 5.5, "w": 1.5},
+                {"x": 11.5, "y": 5.5, "w": 1.5},
+
+                {"x": 13.25, "y": 5.75},
+                {"x": 14.25, "y": 5.75},
+                {"x": 15.25, "y": 5.75}
+            ]
+        },
+        "LAYOUT_ansi_wkl": {
```

Comment:
How do the WKL layouts look on this board? Is there a blocker or there only 2-2 keys on each side of the spacebar?

Reply:
It's the latter. It's 1.5-1.5-7-1.5-1.5 for the bottom row instead of 1.25-1.25-1.25-6.25-1.5-1.5